modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp

```diff
@@ -330,13 +330,14 @@ renderResponse.setTitle(blogsEditEntryDisplayContext.getPageTitle(resourceBundle
 						</aui:fieldset>
 					</liferay-expando:custom-attributes-available>
 
-					<c:if test="<%= (entry == null) || ((entry.getStatus() == WorkflowConstants.STATUS_DRAFT) && entry.isNew()) %>">
-						<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="permissions">
-							<liferay-ui:input-permissions
-								modelName="<%= BlogsEntry.class.getName() %>"
-							/>
-						</aui:fieldset>
-					</c:if>
+				<c:if test="<%= (entry == null) || ((entry.getStatus() == WorkflowConstants.STATUS_DRAFT) && entry.isNew()) %>">
+					<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="permissions">
+						<liferay-ui:input-permissions
+							modelName="<%= BlogsEntry.class.getName() %>"
+						/>
+					</aui:fieldset>
+					<aui:input name="isCreationScreen" value="<%= true %>" type="hidden"/>
```

Comment:
Why the `isCreationScreen` parameter? Why not check for this same condition in the backend (mvc action) just before invoking the service method?